.travis.yml

```diff
@@ -26,23 +26,23 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  matrix: SYMFONY_VERSION=4.0.*
+  matrix: SYMFONY_VERSION=4.2.*
   global:
-    - SYMFONY_DEPRECATIONS_HELPER="/.*each.*/"
+    - SYMFONY_DEPRECATIONS_HELPER="weak"
     - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
-    - SYMFONY_PHPUNIT_VERSION=6
-    - TEST_INSTALLATION=false
+    - KERNEL_CLASS=Symfony\Cmf\Bundle\RoutingBundle\Tests\Fixtures\App\Kernel
```

Comment:
this is no longer needed and does nothing. it is defined in phpunit.xml.dist

Reply:
i removed this line in [#433](https://github.com/symfony-cmf/routing-bundle/pull/433/files#diff-354f30a63fb0907d4ad57269548329e3L33) and the build was green, so i am sure that we can take it away. it should be defined in phpunit.xml.dist, because then it will also work when running the tests locally.

having it here won't break anything, it is simply not seen and can create confusion if somebody needs to change the value and nothing happens when changing it in this file.

Reply:
The var is also in the make file, will remove it there also and we will see :-)

Reply:
thats many places :-/
locally i was not using the makefile but `vendor/bin/phpunit` directly, which worked fine.

Reply:
oh if that works it would be awesome :-)